src/org/labkey/test/tests/list/CrossfolderListTest.java

```diff
@@ -0,0 +1,208 @@
+package org.labkey.test.tests.list;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.data.xml.queryCustomView.FilterType;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.pages.LabkeyErrorPage;
+import org.labkey.test.pages.list.GridPage;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.params.list.IntListDefinition;
+import org.labkey.test.params.list.ListDefinition;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.TestUser;
+import org.labkey.test.util.query.QueryApiHelper;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+@Category({})
```

Comment:
I think we should make this `@Category` align with the `ListTest`. @labkey-tchad would have a better idea of exactly what to set it to. Something like:

```java
@Category({Daily.class, Data.class, Hosting.class})
```